main.go

```diff
@@ -172,7 +172,8 @@ func retryFailedTests(testRunner TestRunner, maxRetries int, timeline *[]api.Tim
 	retries := 0
 	for retries < maxRetries {
 		retries++
-		fmt.Printf("Attempt %d of %d to retry failing tests\n", retries, maxRetries)
+
+		fmt.Printf("+++ Buildkite Test Collector: ♻️ Attempt %d of %d to retry failing tests\n", retries, maxRetries)
```

Comment:
Should this be `Buildkite Test Splitter` like the below?

Reply:
Oh lol